README.md

```diff
@@ -1 +1,3 @@
-# clientk8s
\ No newline at end of file
+[![CircleCI](https://circleci.com/gh/giantswarm/clientk8s.svg?&style=shield)](https://circleci.com/gh/giantswarm/clientk8s)
+
+# clientk8s
```

Comment:
Would be nice to write something in here, along the lines of:

This library provides a set of clients for interacting with k8s resources.
* k8s natives resources (pods, services, nodes, etc ...)
* k8s CRD
* giantswarm custom CRs

Reply:
Added something along these lines to the readme.